app/src/main/java/com/example/learingrealmandretrofit/api/BaseApi.kt

```diff
@@ -4,12 +4,13 @@ import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
 private const val prefix = "api/v1"
-private const val baseUrl: String = "https://word-notes.herokuapp.com"
+private const val baseUrlHeroku: String = "https://word-notes.herokuapp.com"
```

Comment:
You can make 2 flavors of your Android app and select `baseUrl` depending on flavor